### PR TITLE
add a description of the `-color` flag in the manual section of `ocamlc`

### PR DESCRIPTION
**Pull request by c-cube**
Basic doc for the new handling of colors

### CODE REVIEW

**Comment by damiendoligez** [comment]
Would you volunteer to update the man pages too?

**Comment by c-cube** [comment]
Err, I never touched troff before :-S

**Comment by c-cube** [comment]
Well, that is ugly, but it does render more or less properly. Any troff expert to suggest improvements?

File: man/ocamlc.m [hunk 1]
```diff
@@ -259,6 +259,26 @@ Dump detailed information about the compilation (types, bindings,
...
+The following modes are supported:
+
+.B auto
+use heuristics to enable colors only if the output supports them (an ANSI-compatible tty terminal);
+
+.B always
+enable colors unconditionally;
+
+.B never
+disable colors output.
```
Comment (mshinwell) [inline man/ocamlc.m]: s/colors/color

**Comment by gasche** [comment]
Could you rebase the PR against the 4.03 branch?

**Comment by gasche** [comment]
Nevermind, I'll merge.

Rebased and merged: ea967c3499dcdeb3459c857f8999c5c72293480c

**Pull request closed without merging**